src/main/java/edu/kis/powp/jobs2d/drivers/adapters/DrawerDriverAdapter.java

```diff
@@ -26,8 +26,8 @@ public void operateTo(int x, int y) {
 		ILine line = LineFactory.getBasicLine();
 		line.setStartCoordinates(this.startX, this.startY);
 		line.setEndCoordinates(x, y);
-
-		drawLine(line);
+		this.setPosition(x,y);
+		DrawerFeature.getDrawerController().drawLine(line);
```

Comment:
Constructor injection should be used (using a parameter). This is unnecessary direct coupling with DrawerFeature class.